supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalBackend.scala

```diff
@@ -1,29 +1,227 @@
 package cromwell.backend.impl.local
 
-import cromwell.backend.BackendJobExecutionActor.BackendJobExecutionResponse
-import cromwell.backend.BackendLifecycleActor.JobAbortResponse
+import java.nio.file.{FileSystems, Path, Paths}
+
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionAbortedResponse, BackendJobExecutionFailedResponse, BackendJobExecutionResponse, BackendJobExecutionSucceededResponse}
+import cromwell.backend.BackendLifecycleActor.{BackendJobExecutionAbortFailedResponse, BackendJobExecutionAbortSucceededResponse, JobAbortResponse}
 import cromwell.backend._
-import wdl4s.Call
+import cromwell.core.CallContext
+import org.slf4j.LoggerFactory
+import wdl4s._
+import wdl4s.expression.ValueEvaluator
+import wdl4s.values.{WdlFile, WdlValue}
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
 
-class LocalBackend extends BackendJobExecutionActor {
-  /**
-    * Restart or resume a previously-started job.
-    */
-  override def recover: Future[BackendJobExecutionResponse] = ???
+object LocalBackend {
+  val logger = LoggerFactory.getLogger("LocalBackend")
+  // TODO Support GCS ?
+  val fileSystems = List(FileSystems.getDefault)
+
+  private def splitFqn(fullyQualifiedName: FullyQualifiedName): (String, String) = {
+    val lastIndex = fullyQualifiedName.lastIndexOf(".")
+    (fullyQualifiedName.substring(0, lastIndex), fullyQualifiedName.substring(lastIndex + 1))
+  }
+
+  case class Command(argv: Seq[String]) {
+    override def toString = argv.map(s => "\"" + s + "\"").mkString(" ")
+  }
+}
+
+class LocalBackend(override val jobDescriptor: BackendJobDescriptor,
+                   override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendJobExecutionActor with SharedFileSystem {
+
+  import LocalBackend._
+  import better.files._
+  import cromwell.core.PathFactory._
+
+  import scala.sys.process._
+
+  // Mutable process variable assigned when the execute method is called.
+  // Needs to be accessible to be killed when the job is aborted.
+  private var process: Option[Process] = None
```

Comment:
I guess so - we would have to let the backend implementor provide it's own state data ADT etc though.. We can definitely look into it but for this PR it'll be fine I think, especially since abort is not really part of the MVP for this. Also, state data is really a disguised `var` (with more thread-safeness, granted).